### PR TITLE
Add Redundancy Policy

### PR DESCRIPTION
**Pull request by lozette**
Add text for the Redundancy Policy (December 2020) to the "Working Here" section of the playbook. 

Text: https://docs.google.com/document/d/1Tqc9cE1yO43g9_V_eg8bV2VqKZFQCc8NTMyTSOT30Y4/edit

### CODE REVIEW

File: _includes/sections/working-here/redundancy-policy.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+sustainable level. Or that we need less people in a particular specialism if we
+experience a reduction in demand for that type of work.
+
+If a group of people in very similar roles are involved in the redundancy
+situation, everyone in the pool will be treated as being at risk and there will
+be a selection process. We will publish the selection criteria at the time,
+which may include any or all of the following:
+
+- A performance assessment
+- Any disciplinary issues having previously arisen
```
Comment (leeky) [inline _includes/sections/working-here/redundancy-policy.md:41]: "Any past disciplinary issues" ?

Comment (lozette) [reply _includes/sections/working-here/redundancy-policy.md:41]: @Possapostolou has asked to sign off the text as it is for now, hope that's ok

**Review by mattymaxwell — APPROVED** [review]
Approving based on as is

File: _includes/sections/working-here/redundancy-policy.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+markets we serve, could mean that we need to make a role redundant. If that
+happens, we will follow a fair and open process.
+
+This section of the playbook sets out what we’ll do in those circumstances.
+
+#### How roles are selected for redundancy
+
+We will take an open, data-led approach in redundancy situations. We might, for
+example, find that we need to reduce our unbillable roles to keep profits at a
+sustainable level. Or that we need less people in a particular specialism if we
```
Comment (jacksonj04) [inline _includes/sections/working-here/redundancy-policy.md:32]:
```suggestion
sustainable level. Or that we need fewer people in a particular specialism if we
```

**Review by jacksonj04 — COMMENTED** [review]
One "less" vs "fewer" which bugs me, but I don't feel strongly about.

**Pull request merged**